### PR TITLE
Create FT-26-Read-mails

### PR DESCRIPTION
**Pull request by haftus98**
Create epic foler read mail

### CODE REVIEW

**Comment by chibenwa** [comment]
@haftus98 we should avoid flooding the repository with PR. 

I think we can open one PR for BA needs at a single point in time.

File: docs/stories/FT-26-Read-mails [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Basic requirements to read email with JMAP protocol:
+
+ Listing mailbox, thread, emails, ...
+ Receiving mail,
+ Synchronize, refresh,
+ receiving push notification
+ Search email
+ Filter email
```
[the remark this answers is not in the document]
Comment (haftus98) [reply docs/stories/FT-26-Read-mails:8]: Yeah but I’m opening an epic with many other stories in it, should I open an epic folder like this?

[the remark this answers is not in the document]
Comment (chibenwa) [reply docs/stories/FT-26-Read-mails:8]: Let's see this in tomorrow workshop nheeee

[the remark this answers is not in the document]
Comment (hoangdat) [reply docs/stories/FT-26-Read-mails:8]: Each epic only have a ReadMe file:
- Overall information about the epic
- Link to the story, some very short desc is ok
is it ok @dieptran88 ?

**Comment by hoangdat** [comment]
- each story should be in 1 commit 
- each epic should be in 1 PR

**Comment by hoangdat** [comment]
any update for this PR?

**Pull request closed without merging**